airbyte-integrations/connectors/destination-s3/src/main/java/io/airbyte/integrations/destination/s3/parquet/S3ParquetWriter.java

```diff
@@ -56,6 +56,7 @@
 
   private final ParquetWriter<Record> parquetWriter;
   private final AvroRecordFactory avroRecordFactory;
+  public final Schema parquetSchema;
```

Comment:
@tuliren is there a better way to extract the schema from the parquet writer than this?

Reply:
I think this is the right way.

The schema is usually persisted at the beginning of the parquet file, but since we already have it here, we don't need to read it from the file.